semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/Scalacp.scala

```diff
@@ -194,7 +194,15 @@ object Scalacp {
           val stpe = {
             val stag = st.SYMBOL
             val spre = if (tpe.hasNontrivialPrefix) loop(pre) else None
```

Comment:
should we also take care of `hasNonTrivialPrefix`? For the same corner case it currently returns `true` instead of `false`, because the `isObject` test fails.

Reply:
Do you have a test case in mind? Either is now looking good:

```
  symbols {
    symbol: "_empty_.C#Either()."
    kind: METHOD
    properties: 0
    name: "Either"
    tpe {
      tag: METHOD_TYPE
      methodType {
        return_type {
          tag: SINGLETON_TYPE
          singletonType {
            tag: SYMBOL
            symbol: "scala.util.Either."
            primitive: 0
            string: ""
          }
        }
      }
    }
```

Reply:
ah, strange. Given the change, I would expect the prefix to be still there. 👍 then